cocotb/config.py

```diff
@@ -97,6 +99,50 @@ def help_vars_text():
     return helpmsg
 
 
+def lib_name(interface, simulator):
+
+    interface_name = interface.lower()
+    supported_interfaces = ["vpi", "vhpi", "fli"]
+    if interface_name not in supported_interfaces:
+        raise ValueError("Wrong interface used. Supported :" + ", ".join(supported_interfaces))
+
+    simulator_name = simulator.lower()
+    supported_sim = ["icarus", "questa", "modelsim", "ius", "xcelium", "vcs", "ghdl", "riviera", "activehdl", "cvc"]
```

Comment:
```suggestion
    supported_sims = ["icarus", "questa", "modelsim", "ius", "xcelium", "vcs", "ghdl", "riviera", "activehdl", "cvc"]
```
to be consistent with `supported_interfaces`.